app/components/primer/counter_component.rb

```diff
@@ -73,16 +74,16 @@ def value
         "∞"
       else
         if @round
-          count = [@count.to_i, @limit].min
+          count = @has_limit ? [@count.to_i, @limit].min : @count.to_i
           precision = count.between?(100_000, 999_999) ? 0 : 1
           units = {thousand: "k", million: "m", billion: "b"}
           str = number_to_human(count, precision: precision, significant: false, units: units, format: "%n%u")
         else
           @count = @count.to_i
-          str = number_with_delimiter([@count, @limit].min)
+          str = number_with_delimiter(@has_limit ? [@count, @limit].min : @count)
```

Comment:
Since we're changing the logic for the `round: false` code path, might we add a test for it? I see that our only test is for when `round` is `true`.